typescript/sdk/package.json

```diff
@@ -61,7 +61,7 @@
     "prepublishOnly": "yarn build",
     "prettier": "prettier --write ./src",
     "test": "yarn test:unit && yarn test:hardhat && yarn test:foundry",
-    "test:ci": "yarn test:unit --exclude './src/**/SmartProvider.test.ts' && yarn test:hardhat && yarn test:foundry",
+    "test:ci": "yarn test:unit && yarn test:hardhat && yarn test:foundry",
```

Comment:
u can leave it as "test:ci":"yarn test" now :)

Reply:
Good catch, done